gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Gherkin/CommonSteps.cs

```diff
@@ -239,7 +240,8 @@ private static object ToLambda(string stringLambda, string graphName)
 
         private static object ToT(string enumName, string graphName)
```

Comment:
What's the point of the `graphName` argument?

Reply:
It's not really useful to this conversion, but it's the expected signature for the lambda where this method is used. See further up in the definition of `Parsers` where it requires a `Func<string,string,object>` a two arg function basically. maybe we could make that nicer, but this is all plumbing code for tests, so perhaps not a big issue?

Reply:
No, not an issue, I was just curious.